frontend/test/components/filter/pageFilter.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { describe, it, expect } from "vitest";
+import { render, screen } from "@testing-library/vue";
+import PageFilter from "@/components/page/filter/PageFilter.vue";
+
```

Comment:
Okay, I need to go back on what I said originally a little.  There is an appropriate case for a mock function here.  You should create a mock function like:
```typescript
const mockHandleFilterChange = vi.fn();
```
and you'll want to reset it after each test
```typescript
afterEach(() => {
  mockHandleFilterChange.mockClear();
});
```